pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_config_vmware.ts

```diff
@@ -158,15 +158,14 @@ export const vmwareTier0Plan_nfs_cold: PlanData = {
   useExistingStorageMapping: true,
   providerData: vmwareProviderUser,
   networkMappingData: vmwareNetworkMapping_2x_network,
-  storageMappingData: vmwareStorageMapping_nfs,
+  storageMappingData: vmwareStorageMapping_ceph,
   warmMigration: false,
   preHook: preHookData,
   postHook: postHookData,
 };
 
-//Defining vmware cold migration plan for ceph-rbd file system
-export const vmwareTier1Plan_ceph_cold: PlanData = {
-  name: `vmware-tier0-ceph-${vmwareProviderUser.name}-cold`,
+export const vmwareTier1Plan_ceph_cold_duplicate: PlanData = {
+  name: `a-vmware-tier0-ceph-${vmwareProviderUser.name}-cold-duplicate`,
```

Comment:
```suggestion
  name: `a-vmware-tier1-ceph-${vmwareProviderUser.name}-cold-duplicate`,
```
it this `tier0` name on `tier1` test intentional ?

Reply:
It probably makes sense to commit this rename, yes?